packages/api/tests/lambdas/test-kinesis-consumer.js

```diff
@@ -146,9 +153,11 @@ test.serial('A kinesis message, should publish the invalid record to fallbackSNS
     Records: [validRecord, invalidRecord],
   };
   await handler(kinesisEvent, {}, testCallback);
-  const callArgs = publishStub.getCall(0).args;
-  t.deepEqual(invalidRecord, JSON.parse(callArgs[0].Message));
-  t.true(publishStub.calledOnce);
+
+  const publishCalls = snsMock.commandCalls(PublishCommand);
+
+  t.true(publishCalls.length > 0);
```

Comment:
Not sure if it matters too much but should this say `publishCall.length === 1` to better match the previous assertion? There are a few assertions below similar to this one depending on what you decide to do with it.

Reply:
Yeah, makes sense. The half-baked idea was to future-proof the test because we really only care that the publish command is called, not how many times (if we were to add another call down the line). But... that doesn't make sense. Better to be explicit and guard against some accidental call that might be introduced. A change in behavior should require a test update.